PWGHF/D2H/Tasks/taskB0.cxx

```diff
@@ -221,6 +246,16 @@ struct HfTaskB0 {
         registry.fill(HIST("hCPADRecBg"), candD.cpa(), ptCandB0);
         registry.fill(HIST("hDecLengthDRecBg"), candD.decayLength(), ptCandB0);
         registry.fill(HIST("hChi2PCARecBg"), candidate.chi2PCA(), ptCandB0);
+
+        if (checkDecayTypeMc) {
+          if (TESTBIT(flagMcMatchRecB0, DecayTypeMc::B0ToDsPiAndDsToKKPi)) { // B0 → Ds- π+ → (K- K+ π-) π+
+            registry.fill(HIST("hDecayTypeMc"), 1 + DecayTypeMc::B0ToDsPiAndDsToKKPi, invMassCandB0, ptCandB0);
+          } else if (TESTBIT(flagMcMatchRecB0, DecayTypeMc::PartlyRecoDecay)) { // Partly reconstructed decay channel
+            registry.fill(HIST("hDecayTypeMc"), 1 + DecayTypeMc::PartlyRecoDecay, invMassCandB0, ptCandB0);
+          } else {
+            registry.fill(HIST("hDecayTypeMc"), 1 + OtherDecay, invMassCandB0, ptCandB0);
```

Comment:
```suggestion
            registry.fill(HIST("hDecayTypeMc"), 1 + DecayTypeMc::OtherDecay, invMassCandB0, ptCandB0);
```
In general, stick to one convention: `DecayTypeMc::XXX` or `hf_cand_b0::DecayTypeMc::XXX`